changelog/changes/teach-stairs-minute.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Dynamic `log_type` for `to_google_secops`"
+type: feature
+authors: raxyte
+pr: 5365
+---
+
+The `to_google_secops` operator now supports dynamic `log_type`s. The option can
+now be set to any expression evaluating to a string, e.g.
```

Comment:
Please fix passive voice and punctation (i.e., a missing colon after `e.g.`). I would always let AI write at this point to save cognitive bandwidth.

Reply:
This is already active voice.